go-fuzz/hub.go

```diff
@@ -116,6 +118,24 @@ func newHub(metadata MetaData) *Hub {
 			ro.intLits = append(ro.intLits, []byte(lit.Val))
 		}
 	}
+	if *flagDict != "" {
+		ro.strLits = nil // Discard existing tokens
+		fileName := *flagDict
```

Comment:
This renaming looks excessive, especially if we remove the log.Printf. Just use *flagDict.